app/src/main/java/hm/orz/chaos114/android/tumekyouen/util/ServerUtil.java

```diff
@@ -29,251 +19,261 @@
+                                  final String tokenSecret) throws IOException {
+        String url = context.getString(R.string.server_url) + "/page/api_login";
+        Map<String, String> params = new HashMap<String, String>();
+        params.put("token", token);
+        params.put("token_secret", tokenSecret);
 
-		post(url, params);
-	}
+        post(url, params);
+    }
 
-	/**
-	 * ステージクリア情報を送信する。
-	 * 
-	 * @param context コンテキスト
-	 * @param stageModel クリアステージ情報
-	 */
-	public static void addStageUser(final Context context,
-			final TumeKyouenModel stageModel) {
-		String url = context.getString(R.string.server_url) + "/page/add";
-		Map<String, String> params = new HashMap<String, String>();
-		params.put("stageNo", Integer.toString(stageModel.getStageNo()));
+    /**
+     * ステージクリア情報を送信する。
+     *
+     * @param context    コンテキスト
+     * @param stageModel クリアステージ情報
+     */
+    public static void addStageUser(final Context context,
+                                    final TumeKyouenModel stageModel) {
+        String url = context.getString(R.string.server_url) + "/page/add";
+        Map<String, String> params = new HashMap<String, String>();
+        params.put("stageNo", Integer.toString(stageModel.getStageNo()));
 
-		try {
-			post(url, params);
-		} catch (Exception e) {
-			Log.e(TAG, "クリア情報の送信に失敗", e);
-		}
-	}
+        try {
+            post(url, params);
+        } catch (Exception e) {
+            Log.e(TAG, "クリア情報の送信に失敗", e);
+        }
+    }
 
-	/**
-	 * すべてのデータを送信する。
-	 * 
-	 * @param context コンテキスト
-	 * @param stages クリアステージ情報リスト
-	 * @return 返却JSON
-	 */
-	public static List<TumeKyouenModel> addAllStageUser(final Context context,
-			List<TumeKyouenModel> stages) {
-		// 全ステージを送信する
-		String url = context.getString(R.string.server_url) + "/page/add_all";
-		JSONArray sendData = new JSONArray();
-		DateFormat simpleDateFormat = new SimpleDateFormat(
-				"yyyy-MM-dd HH:mm:ss");
-		simpleDateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
-		for (TumeKyouenModel stageModel : stages) {
-			JSONObject map = new JSONObject();
-			try {
-				map.put("stageNo", Integer.toString(stageModel.getStageNo()));
-				map.put("clearDate",
-						simpleDateFormat.format(stageModel.getClearDate()));
-			} catch (JSONException e) {
-				continue;
-			}
-			sendData.put(map);
-		}
-		Map<String, String> params = new HashMap<String, String>();
-		params.put("data", sendData.toString());
+    /**
+     * すべてのデータを送信する。
+     *
+     * @param context コンテキスト
+     * @param stages  クリアステージ情報リスト
+     * @return 返却JSON
+     */
+    public static List<TumeKyouenModel> addAllStageUser(final Context context,
+                                                        List<TumeKyouenModel> stages) {
+        // 全ステージを送信する
+        String url = context.getString(R.string.server_url) + "/page/add_all";
+        JSONArray sendData = new JSONArray();
+        DateFormat simpleDateFormat = new SimpleDateFormat(
```

Comment:
WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.